multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java

```diff
@@ -45,11 +45,11 @@
     @Inject
     protected ProcessTypeParser processTypeParser;
     @Autowired(required = false)
-    private ProcessTypeToOperationMetadataMapper operationMetadataMapper;
+    private transient ProcessTypeToOperationMetadataMapper operationMetadataMapper;
```

Comment:
why transient is needed at all? and why only to operationMetadataMapper and currentTimeSupplier

Reply:
Don't know. Sonar complains about only these two